test/OrchardCore.Commerce.Tests.UI/Tests/TieredPricePartTests/PersistenceTieredPriceTests.cs

```diff
@@ -0,0 +1,68 @@
+using Atata;
+using Lombiq.Tests.UI.Attributes;
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Services;
+using OpenQA.Selenium;
+using OrchardCore.Commerce.MoneyDataType;
+using Shouldly;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace OrchardCore.Commerce.Tests.UI.Tests.TieredPricePartTests;
+
+public class PersistenceTieredPriceTests : UITestBase
+{
+    public PersistenceTieredPriceTests(ITestOutputHelper testOutputHelper)
+        : base(testOutputHelper)
+    {
+    }
+
+    [Theory, Chrome]
+    public Task CreatingNewTieredPriceShouldPersist(Browser browser) =>
+        ExecuteTestAfterSetupAsync(
+            async context =>
+            {
+                await context.SignInDirectlyAsync();
+                await context.CreateNewContentItemAsync("TestTieredPriceProduct");
+
+                const string sku = "UITESTSKU"; // #spell-check-ignore-line
+                const string price = "9999";
+                const string quantity1 = "2";
+                const string unitPrice1 = "8999";
+                const string quantity2 = "5";
+                const string unitPrice2 = "7999";
+                const string addPriceTierButtonXPath = "//div[contains(@class, 'add-price-tier-button')]";
```

Comment:
Instead of XPath you can store only the class name and then use `By.ClassName()`